process/css-styleguide/button/button.css

```diff
@@ -0,0 +1,183 @@
+@import "../default.css";
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0.625rem 1.25rem;
+  gap: 0.563rem;
+  width: 4.9rem;
+  height: 2.37rem;
+  font-size: 0.75rem;
+  border-radius: 11rem;
+  border-width: 0.1rem;
+  border-style: solid;
+  padding: 0.75rem 1.5rem;
+  cursor: pointer;
+  outline: none;
+  transition: 0.3s ease;
+  transition-property: background-color, border-color;
+  color: var(--white);
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+}
+
+.btn-lg {
+  padding: 0.75rem 1.5rem;
+  gap: 0.75rem;
+  width: 5.875rem;
+  height: 2.813rem;
+  font-size: 0.875rem;
+}
+
+.btn-sm {
+  padding: 0.313rem 0.75rem;
+  gap: 0.438rem;
+  width: 3.938rem;
+  height: 1.75rem;
+  font-size: 0.75rem;
+}
+
+.btn-dodger-blue {
+  background-color: var(--dodger-blue-500);
+  border-color: var(--dodger-blue-500);
+}
+
+.btn-hot-pink {
+  background-color: var(--hot-pink-500);
+  border-color: var(--hot-pink-500);
+}
+
+.btn-lila {
+  background-color: var(--lila-500);
+  border-color: var(--lila-500);
+}
+
+.btn-turquoise {
+  background-color: var(--turquoise-600);
+  border-color: var(--turquoise-600);
+}
+
+.btn-watermelon {
+  background-color: var(--watermelon-500);
+  border-color: var(--watermelon-500);
+}
+
+.btn-peppermint {
+  background-color: var(--peppermint-500);
+  border-color: var(--peppermint-500);
+}
+
+.btn-candlelight {
+  background-color: var(--candlelight-600);
+  border-color: var(--candlelight-600);
+}
+
+.btn:disabled {
+  background-color: var(--ashes-900);
+  border-color: var(--ashes-900);
+  pointer-events: none;
+}
+
+.btn-dodger-blue:hover,
+.btn-dodger-blue:focus {
+  background-color: var(--dodger-blue-600);
+  border-color: var(--dodger-blue-600);
+  color: var(--white);
+}

```

Comment:
Some buttons have a not-that-obvious contrast between the default state and the hover/focus one. 
I'm thinking of the following colors:
- dodgerBlue
- hotPink
- watermelon

Maybe we can use the `700` color variant for them in order to increase the contrast and have better accessibility

Reply:
I followed the colors specified in the design system. I checked with @lpinot, and she thinks we should stick with what's defined there for consistency reasons. Those are the colors used in the landing too.